### PR TITLE
[Xray] Release xray 2.13.0

### PR DESCRIPTION
**Pull request by rahulsadanandan**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/xray/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: xray
-version: 2.0.2
-appVersion: 2.12.1
+version: 2.0.3
```
Comment (eldada) [inline stable/xray/Chart.yaml:3]: Should be 2.1.0 as xray version also has a new minor versiom.

Comment (danielezer) [reply stable/xray/Chart.yaml:3]: +1

File: stable/xray/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ helm repo add jfrog https://charts.jfrog.io
 ### Install Chart
 Install JFrog Xray
 ```bash
-helm upgrade --install xray --namespace xray jfrog/xray --version 2.0.2
```
Comment (chukka) [inline stable/xray/README.md:33]: @rahulsadanandan please change to `2.1.0` here too

File: stable/xray/README.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ helm repo add jfrog https://charts.jfrog.io
...
+helm upgrade --install xray --namespace xray jfrog/xray --version 2.0.3
```
Comment (danielezer) [inline stable/xray/README.md:33]: @rahulsadanandan Please change the README version to 2.1.0 as well

File: stable/xray/README.md (hunk 2 above)
Comment (rahulsadanandan) [reply stable/xray/README.md:33]: Changed now

File: stable/xray/Chart.yaml (hunk 1 above)
Comment (rahulsadanandan) [reply stable/xray/Chart.yaml]: Corrected now.

**Pull request merged**